step-generation/src/utils/pythonFileUtils.ts

```diff
@@ -620,3 +628,45 @@ export const formatChangeTipArg = (changeTip: ChangeTipOptions): string => {
     }
   }
 }
+export const getSetStoredLabware = (
+  moduleEntities: ModuleEntities,
+  labwareEntities: LabwareEntities,
+  labware: { [labwareId: string]: LabwareTemporalProperties }
+): string => {
+  const pythonSetStoredLabware = Object.values(moduleEntities).map(module => {
+    const { id, type, pythonName } = module
+
+    if (type === FLEX_STACKER_MODULE_TYPE) {
+      const labwareOnModule = Object.entries(labware).find(([_, labware]) =>
+        labware.stack?.includes(id)
+      )
+      if (labwareOnModule == null) {
+        return ''
+      } else {
+        // Count only labware items (excluding slot, module, and adapters)
+        const labwareCount = labwareOnModule[1].stack.filter(
+          itemId =>
+            itemId in labwareEntities &&
+            !labwareEntities[itemId].def.allowedRoles?.includes('adapter') &&
+            !labwareEntities[itemId].def.allowedRoles?.includes('lid')
+        ).length
+        const labwareEntity = labwareEntities[labwareOnModule[0]]
+        const setStoredLabwareArgs = [
+          `\nloadName=${formatPyStr(labwareEntity.def.parameters.loadName)}`,
+          `namespace=${formatPyStr(labwareEntity.def.namespace)}`,
+          `version=${labwareEntity.def.version}`,
+          `count=${labwareCount}`,
+        ].join(',\n')
+        console.log('setStoredLabwareArgs: ', setStoredLabwareArgs)
+        console.log(
+          'set stored labware command: ',
+          `${pythonName} = ${PROTOCOL_CONTEXT_NAME}.set_stored_labware(${indentPyLines(setStoredLabwareArgs)})`
+        )
+        return `${pythonName} = ${PROTOCOL_CONTEXT_NAME}.set_stored_labware(${indentPyLines(setStoredLabwareArgs)})`
```

Comment:
this should indent on the right line 

```suggestion
        return `${pythonName} = ${PROTOCOL_CONTEXT_NAME}.set_stored_labware(\n${indentPyLines(setStoredLabwareArgs)})`
```